server/src/main/scala/controllers/application/ShiftsController.scala

```diff
@@ -178,11 +166,53 @@ class ShiftsController @Inject()(cc: ControllerComponents,
     }
   }
 
+  // It is different from createShifts as here we need to check if the shift already exists
```

Comment:
I understand that there are more checks to handle when adding shifts compared to creating a first shift, but those checks can surely be made when creating the first shifts too? It would be so much cleaner to have a single interface to save shifts than handling 2 different workflows from the front end all the way to the back end. 

The front end espectially should not have to know if it's asking to save a first shift or an additional shift. It should just say 'save these shifts' and the backend handles it accordingly.

The point is, this function can also be used for saving shifts. It doesn't matter that it's checking for overlapping shifts or ones with the same name. It just won't find any when creating the first shifts